src/Standards/PSR12/Tests/Methods/NullableWhitespaceUnitTest.inc

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
* I don't see any tests for nullable parameter type declarations (can be combined with the existing tests).
* Why do you have separate test for `interface`s ? You are sniffing for `T_NULLABLE`, not for `class`/`interface` or `trait` for that matter, so the test is not testing anything new.
    May be it is an idea to mix the tests up a bit: have one test with a global function, one with a closure, one with a method in a class, one with an interface method etc ?